src/hooks/useTranslation.tsx

```diff
@@ -60,6 +60,7 @@ const translationMap = {
 	'placeholders.select_edition': 'Select edition',
 
 	'labels.instructions': 'Instructions',
+	'labels.provisioning_key_name': 'Provisioning Key name: ',
```

Comment:
@nitishagar It might make more sense to have it w/o the `:` (and the extra empty space) like: `Provisioning Key name`
given that none of the rest labels have a `:`.

Reply:
@thgreasi Updated here: https://github.com/balena-io-modules/rendition/pull/1490